frontend/public/components/utils/link.tsx

```diff
@@ -84,9 +83,17 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
 );
 
 // Open links in a new window and set noopener/noreferrer.
-export const LinkifyExternal: React.FC<{ children: React.ReactNode }> = ({ children }) => (
-  <Linkify properties={{ target: '_blank', rel: 'noopener noreferrer' }}>{children}</Linkify>
-);
+export const LinkifyExternal: React.FC<LinkifyExternalProps> = ({
+  children,
+  fuzzyEmail = true, // recognize emails without mailto: prefix. Default true.
+  fuzzyIP = false, // allow IPs in fuzzy links above. Can conflict with some texts like version numbers. Default false.
+  fuzzyLink = true, // recognize URL-s without http(s):// head. Default true.
+}) => {
+  linkify.set({ fuzzyEmail, fuzzyIP, fuzzyLink });
```

Comment:
I don't think this will work correctly. This is setting the global configuration. If there are two components with different configuration, you don't know what you'll get.